api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -90,17 +89,15 @@ public void bufferBillingProjects() {
 
   @Override
   public Collection<MeasurementBundle> getGaugeData() {
-    final ImmutableMap<BufferEntryStatus, Long> entryStatusToCount =
-        ImmutableMap.copyOf(billingProjectBufferEntryDao.getCountByStatusMap());
-
-    return Arrays.stream(BufferEntryStatus.values())
```

Comment:
previously would cycle through all status, outputting 0 if none are found; this is not necessary.